credits.md

```diff
@@ -14,3 +14,4 @@
 - Labeled Toggle - [Aiko](https://twitter.com/username_aiko)
 - Border Glow Button - [Ethan Pollack](https://epoll31.github.io)
 - Shine Button - [Shriprasanna](https://twitter.com/shriprasanna007)
+- Text Revel Button - [Shriprasanna](https://twitter.com/shriprasanna007)
```

Comment:
Spelling Mistake here, should be `TextReveal`

Reply:
fixed